tutorials/10-configuration.md

```diff
@@ -1,11 +1,11 @@
 Configuration
 ==============
 
-All configuration is done in config/config.json or with equivalent environment variables (see [sample.env](./sample.env)). **Leave config/default-config.json unchanged.** Whichever of these 2 methods you choose, will override the defaults set in config/default-config.json.
+All configuration is done in config/config.json or with equivalent environment variables (see [sample.env](https://github.com/enketo/enketo-express/blob/master/config/sample.env)). **Leave config/default-config.json unchanged.** Whichever of these 2 methods you choose, will override the defaults set in config/default-config.json.
 
 Below is a complete list of all configuration items. The **bold items are important to set**. Others are less important.
 
-#### app name 
+#### app name
```

Comment:
Can these configuration items (or just all ### headers) get a bookmark (i.e. a id attribute)? We're often pointing users to the documentation of a particular config item. For this one `id="app-name"` as in https://github.com/enketo/enketo-express/blob/master/config/README.md#app-name

Reply:
You can do it with this JSDoc already - all the headers automatically get `id`s (e.g. https://enketo.github.io/enketo-core/tutorial-20-development.html#notes-for-css-developers). The only problem is that you need to get it from inspecting the code, as there are no fancy anchor icons as you usually have.

Reply:
Ah right. That will do for now!